common/flatpak-dir-private.h

```diff
@@ -71,6 +71,7 @@ GType flatpak_deploy_get_type (void);
 #define FLATPAK_REPO_DESCRIPTION_KEY "Description"
 #define FLATPAK_REPO_HOMEPAGE_KEY "Homepage"
 #define FLATPAK_REPO_ICON_KEY "Icon"
+#define FLATPAK_REPO_FILTER_KEY "Filter"
```

Comment:
We have a man page for the flatpakrepo file format. Should update it to mention the Filter key, and its special handling.